config/routes.php

```diff
@@ -44,6 +44,8 @@
 Router::defaultRouteClass(DashedRoute::class);
 
 Router::scope('/', function (RouteBuilder $routes) {
+    /* @var \Cake\Routing\RouteBuilder $routes */
```

Comment:
No, I just find it funny that the variable is type hinted but you sill need to add another hint in th next line